### ISSUE TITLE
Input doesn't work in Wolfenstein 3d N64 port

### ISSUE THREAD

**Issue by fzurita** [mupen64plus/mupen64plus-core#838]
As the title says, in the homebrew Wolfenstein 3d N64 port, controller input doesn't work. From what I have read online, that port works ok in the real console.

**Comment by loganmc10** [comment]
I couldn't reproduce this, this is the ROM I found:
```
Core: Name: Wolfenstein 64      
Core: MD5: 222F5623D55A1BD5D02F9220FC673E57
Core: CRC: 6676A289 39687730
```
The controls worked, although it doesn't support the analog stick, it's a d-pad game. There was no audio, but maybe the game just doesn't have audio? It is a homebrew.

I had to use the Pure Interpreter to launch it, I suspect it was made using libdragon and has this issue: https://github.com/mupen64plus/mupen64plus-core/issues/738
